app/src/docs/v1.api-spec.yaml

```diff
@@ -345,16 +539,22 @@ components:
         body:
           type: string
           description: >-
-            A Nunjucks compatible body template of the message as an Unicode
-            string, Buffer, Stream or an attachment-like object. Refer to
-            https://mozilla.github.io/nunjucks/templating.html for template
-            syntax.
-          example: '{{ something.greeting }} {{ something.target }} content'
-        contexts:
+            The body of the message as an Unicode string, Buffer, Stream or an
```

Comment:
same as above, let's just say it is a string.

Reply:
Dropped the sentence to just say unicode string for now.